src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -83,9 +97,16 @@ def load_arguments(self, _):
 
     with self.argument_context('backup policy set') as c:
         c.argument('policy', type=file_type, help='JSON encoded policy definition. Use the show command with JSON output to obtain a policy object. Modify the values using a file editor and pass the object.', completer=FilesCompleter())
+        c.argument('name', options_list=['--name', '-n'], help='Name of the Policy.')
```

Comment:
--name is user defined name for above --policy? Can you refine your help message for easy understanding?

Reply:
@pvrk

Reply:
Yes, name is user defined name for policy. This is the same name that appears when you retrieve in Azure Portal/PS and REST API. So this is quite familiar to our users. May I know what further clarification are you looking for?

Reply:
Thanks for you clarification. I am just confused with --name and --policy. One more question is why --name is not required? In this way, how do you know which policy to update?